indra/resources/source_info.json

```
@@ -160,5 +160,11 @@
         "link": "https://github.com/chemical-roles/chemical-roles",
         "type": "database",
         "domain": "biology"
+    },
+    "conib": {
+        "name": "CONIB",
+        "link": "https://pharmacome.github.io/conib/",
```

maybe better to link to the github repo?

```suggestion
        "link": "https://github.com/pharmacome/conib",
```